draft-ietf-quic-transport.md

```diff
@@ -4554,9 +4554,11 @@ connection IDs gives clients some assurance that the server received the packet
 and that the Version Negotiation packet was not generated by an off-path
 attacker.
 
-As future versions of QUIC may support Connection IDs larger than the version 1
-limit, Version Negotiation packets could carry Connection IDs that are longer
-than 20 bytes.
+Future versions of QUIC may have different requirements for the lengths of
+connection IDs. In particular, connection IDs might have a smaller minimum
+length or a greater maximum length.  Version-specific rules for QUIC packets
```

Comment:
With the context of the first new sentence the second sentence is clearly apply to CIDs, but it indicates that there might be other aspects. What aspects are not invariant here?